api/swagger-spec/v1.json

```diff
@@ -17624,6 +17628,40 @@
      }
     }
    },
+   "v1.AzureDiskVolumeSource": {
+    "id": "v1.AzureDiskVolumeSource",
+    "description": "AzureDisk represents an Azure Data Disk mount on the host and bind mount to the pod.",
+    "required": [
+     "diskName",
```

Comment:
yes, both diskName and ".vhd" suffix appear to be required or Azure API returns InvalidParameter error.